source/common/stats/symbol_table_impl.cc

```diff
@@ -128,8 +127,7 @@ SymbolVec SymbolTableImpl::Encoding::decodeSymbols(const SymbolTable::Storage ar
 }
 
 void SymbolTableImpl::Encoding::decodeTokens(
-    const SymbolTable::Storage array, uint64_t size,
-    const std::function<void(Symbol)>& symbolTokenFn,
+    const SymbolTable::Storage array, size_t size, const std::function<void(Symbol)>& symbolTokenFn,
```

Comment:
not your fault, but clang-tidy is dinging you for a pre-existing naming mistake here.

s/symbolTokenFn/symbol_token_fn/ here and in the header. Sorry about that.

That will help you pass CI.